sources/web/datalab/static/datalab.css

```diff
@@ -215,14 +215,14 @@ div.btn-group a {
 div.btn-toolbar {
   margin: 0px;
 }
-div.btn-toolbar .toolbar-btn {
+div.btn-toolbar .toolbar-btn, #help_menu_container .dropdown-toggle {
   background-color: transparent;
   border: none;
   font-family: 'Open Sans', Helvetica, sans-serif;
   font-weight: normal;
   font-size: 14px;
   outline: solid 1px transparent !important;
-  height: 46px;
+  /* height: 46px; */
```

Comment:
I left this in in hopes someone else might make an alternate suggestion.

At this point it looks like we're not going to merge this branch, so I will wait until later (if ever) to clean it up.